File: C/lt_01_estseq/lt01_estseq_42.c
#include <stdio.h>

int main() {
  float top = 1, bottom = 1;
  int num, i;
  double soma = 0.0;

  printf("Insira o numero de elementos da serie: ");
  scanf("%d", &num);

  for (i = 0; i < num; i++) {
    soma += top / bottom;
    top += 1;
    bottom += 2;
  }

  printf("O valor %d da serie sera %.2lf.\n", num, soma);
  return 0;
}

File: rust/lt01-estseq/lt01-estseq-42.rs
fn main() {
    let mut i = 1.0;
    let mut j = 1.0;
    let mut soma = 0.0;
    while i <= 64.0 {
        soma += i / j;
        i += 1.0;
        j += 2.0;
    }
    println!("O valor da serie é {:.2}.", soma);
}
